synapse/__init__.py

```diff
@@ -17,8 +17,11 @@
 """ This is a reference implementation of a Matrix home server.
 """
 
+import os
 import sys
 
+from tests.patch_inline_callbacks import do_patch
```

Comment:
let's not import test code into the main source - can you move the patcher into the main synapse codebase?